<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Voiture</class>
 <widget class="QDialog" name="Voiture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>524</width>
     <height>277</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Disponible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView_dispo"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_modifier">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_reserver">
          <property name="text">
           <string>Reserver</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_supprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Reserv√©</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView_reserv"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
